rfcs/RFC-Upload-From-Dropbox.md

```diff
@@ -0,0 +1,358 @@
+# User Uploads From Drop Box
```

Comment:
nit on terminology.  In most cases you use "upload", but there are many cases where "download" is used for the same thing.   It doesn't matter too much for the document because it's clear what is meant.  But in the code we will want to be consistent.  E.g.  there is currently Download Queue, Download Job Entry, Upload Table, Upload Service, ..

Reply:
I'll change all to be upload except when referring to downloading from DropBox